src/iso15118/tbd_controller.cpp

```diff
@@ -87,7 +91,11 @@ void TbdController::handle_sdp_server_input() {
 
     auto connection = [this](bool secure_connection) -> std::unique_ptr<io::IConnection> {
         if (secure_connection) {
+#ifdef LIBISO_OPENSSL
             return std::make_unique<io::ConnectionSSL>(poll_manager, config.interface_name, config.ssl);
```

Comment:
this #ifdef could be removed if a using statement was used after lines 12 and 14
e.g. "using ConnectionType = io::ConnectionSSL;" and "using ConnectionType = io::ConnectionTLS;"
return std::make_unique<ConnectionType>(poll_manager, config.interface_name, config.ssl);

Reply:
Good suggestion! I'll change it like this